### ISSUE TITLE
Auto Select Profile on startup based on game (BMS or DCS)

### ISSUE THREAD

**Issue by johndcollins** [HeliosVirtualCockpit/Helios#623]
I have a second computer running my MFDs and I'd like to run either DCS F/16 or Falcon BMS. 
My MFD computer doesn't get started until I flip the MFD switch on in my pit, so I'd like to have Helios control center auto select my profile based on the simulator I'm currently running. I see that the DCS Interface has a active vehicle hint, but this doesn't work with Falcon BMS I'd assume.
Is there a way to have the control center pick which profile to run based on the simulator running on the main computer? I'd hate to have to turn on my MFD computer every time I want to switch games and select the default profile on control center.

**Comment by Todd1215** [comment]
Control Center can auto start a profile but this is currently limited to DCS. The functionality hasn't been extended to BMS as it relies on the type of aircraft to associate with the profile. Its not specific to what executable is running as you could fly different aircraft and each have its own profile.

How it would work is that once BMS commits to 3D, only then would it select the correct profile as that is the only point that the sharedmemory data for an aircraft is available. So your second PC would need to mirror the sharedmemory of the main PC where BMS is running to even be able to determine which nomenclature airframe the pilot is in within 3D.

**Comment by johndcollins** [comment]
Ok, but this is not the functionality now. So this is missing functionality. Just want to make sure I'm not misunderstanding you comment. What you are saying is how it could work?

**Comment by Todd1215** [comment]
Yes, this functionality only exists for dcs currently but not based on process running.

**Comment by johndcollins** [comment]
With the RTT client sharememory how hard would this be to implement do you think? 
Could the control center not just listen to the RTT server to see if BMS is running, then switch to the BMS profile?

**Comment by Todd1215** [comment]
I think the best aproach is to be able to auto start a profile from the command line. This is currently an option but all it does at the moment is start control center and set the profile that is passed on the commandline. You can also kill the control center with a commandline. So building a simple script to monitor a process and launch controlcenter with options is probably the best approach. I need to look into why the autostart profile option for the commandline doesn't actually start the profile.

**Comment by johndcollins** [comment]
Thanks, I will look into that.
